cmd/operator-opamp-bridge/config/config.go

```diff
@@ -18,14 +18,14 @@ import (
 	"crypto/rand"
 	"errors"
 	"fmt"
+	"github.com/oklog/ulid"
```

Comment:
This import is in the wrong place

Reply:
I think we also probs want the v2

Reply:
yeah this was some goland side effect i didnt intend to change, i reverted that bit